pkg/graveler/ref/manager_test.go

```diff
@@ -19,625 +19,655 @@ import (
 )
 
 func TestManager_GetRepository(t *testing.T) {
-	r := testRefManager(t)
-	t.Run("repo_doesnt_exist", func(t *testing.T) {
-		_, err := r.GetRepository(context.Background(), "example-repo")
-		if !errors.Is(err, graveler.ErrRepositoryNotFound) {
-			t.Fatalf("expected ErrRepositoryNotFound got error: %v", err)
-		}
-	})
-	t.Run("repo_exists", func(t *testing.T) {
-		repoID := graveler.RepositoryID("example-repo")
-		branchID := graveler.BranchID("weird-branch")
-
-		testutil.Must(t, r.CreateRepository(context.Background(), repoID, graveler.Repository{
-			StorageNamespace: "s3://foo",
-			CreationDate:     time.Now(),
-			DefaultBranchID:  branchID,
-		}, ""))
-
-		repo, err := r.GetRepository(context.Background(), repoID)
-		if err != nil {
-			t.Fatalf("unexpected error: %v", err)
-		}
+	tests := testRefManager(t)
+	for _, tt := range tests {
+		t.Run("repo_doesnt_exist", func(t *testing.T) {
```

Comment:
here and elsewhere - I think the name of the ref is critical for debugging the right implementation.

Reply:
done

Reply:
Still not seeing the type of the ref manager in the test name

Reply:
done